.github/get-modified-packages.php

```diff
@@ -22,7 +22,7 @@ function getPackageType(string $packageDir): string
     return match (true) {
         str_contains($packageDir, 'Symfony/Bridge/') => 'bridge',
         str_contains($packageDir, 'Symfony/Bundle/') => 'bundle',
-        preg_match('@Symfony/Component/[^/]+/Bridge/@', $packageDir) => 'component_bridge',
+        1 === preg_match('@Symfony/Component/[^/]+/Bridge/@', $packageDir) => 'component_bridge',
```

Comment:
> Yay! Package checks are passing, the match case for component_bridge wasn't behaving as expected, back onto PHP tests 😄

Thank you for pointing that out. 

I opened a PR to backport this fix into 7.2: https://github.com/symfony/symfony/pull/60216